features/with.yml.dist

```diff
@@ -0,0 +1,50 @@
+# Generated from: with.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+status:
+  baseline: false
+  support:
+    chrome: "38"
+    chrome_android: "38"
+    edge: "12"
+    firefox: "48"
+    firefox_android: "48"
+    safari: "10"
+    safari_ios: "10"
```

Comment:
Given the current Baseline definition, this is correct. That said, I expect consumers (like MDN and caniuse) to choose to suppress the Baseline information when there's a discouragement.